Sources/NativeBuilder/ContainerBuildIR/Graph/GraphBuilder+Arg.swift

```diff
@@ -0,0 +1,99 @@
+//===----------------------------------------------------------------------===//
+// Copyright © 2025 Apple Inc. and the container project authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//===----------------------------------------------------------------------===//
+
+import Foundation
+
+extension GraphBuilder {
+    /// Add a build argument.
+    /// These arguments are available within the stage.
+    @discardableResult
+    public func arg(_ name: String, defaultValue: String? = nil) throws -> Self {
+        let operation = MetadataOperation(
+            action: .declareArg(name: name, defaultValue: defaultValue)
+        )
+        return try add(operation)
+    }
+
+    /// Add a FROM-only build argument (ARG before the first FROM).
```

Comment:
Buildkit calls them `meta args`. For parity, it would be good to call them the same here